### ISSUE TITLE
Swapping bars when Shield is equipped

### ISSUE THREAD

**Issue by d87** [tullamods/Dominos#449]
**Is your feature request related to a problem? Please describe.**
In Shadowlands DPS warriors and paladins get abilities like  shield of the righeousness and shield block that require shield to be equipped.

**Describe the solution you'd like**
A fake 'Defensive stance' bar swapping based on [equipped:Shields] conditional. And it's only for Arms, Fury and Retribution

btw the weapon types in this conditional is locale-dependent and the best way i know to query the translated shields name is ```select(3,GetItemInfoInstant(32375))```

**Comment by Tuller** [comment]
This should be implemented in the latest 9.0.0-beta3.  This is what you can use to get Shields back from the API:
```lua
local name = GetItemSubClassInfo(LE_ITEM_CLASS_ARMOR, LE_ITEM_ARMOR_SHIELD)
```

**Comment by d87** [comment]
Thanks!
